lib/index.coffee

```diff
@@ -10,13 +10,24 @@ dockerUtils = require './docker'
 
 class DockerGC
 	setDocker: (hostObj) ->
+		@baseImage = null
 		@currentMtimes = {}
 		@hostObj = _.defaults({ Promise }, hostObj)
 		@dockerProgress = new DockerProgress(@hostObj)
 		dockerUtils.getDocker(@hostObj)
 		.then (docker) =>
 			@docker = docker
 
+			# Docker info can take a while so do it here,
+			# and don't wait on the results
+			@getDaemonArchitecture()
+			.then (arch) =>
```

Comment:
This can be unindented

Reply:
It can't, becase it relies on the `@docker` variable from above, e.g. we must have first assigned it.

Reply:
Where within this `.then` does `@docker` get used? (Besides that because it's a `.then` it will happen in order so even if it did use it then it would still happen after it was assigned)